extra_docs/relations/README.md

```diff
@@ -156,16 +161,118 @@ dom.dataset("posts")
   .execute()
 ```
 
-### [Attach and detach records](#attachdetach)
+### [Attach and detach records](#attach-and-detach-records)
 
-We were describing how to insert parent and child records in one go, but often we need to add related records to the 
-existent data. In our schema it would be adding a comment to the certain post. 
+We have described how to insert parent and child records in one go, but eventually, we need to relate existent records
+to other ones. You can do it using `.attach()` and `.detach()`.
+
+Let's say you have inserted a post:
+
+```typescript
+  const posts = await dom.dataset("posts")
+    .insert({
+      title: "A post with no comments",
+      message: "Okay",
+    })
+    .execute();
+
+  console.log(posts);
+  // [
+  //   {
+  //     id: ...,
+  //     title: "A post with no comments",
+  //     message: "Okay",
+  //   }
+  // ]
+```
+
+Then some comments...
+
+```typescript
+  const comments = await dom.dataset("comments")
+    .insert([
+      {
+        text: "Very nice, congrats!",
+        like: true,
+      },
+      {
+        text: "Might've been waaaay better",
+        like: false,
+      },
+    ])
+    .execute();
+
+  console.log(comments);
+  // [
+  //   { id: ..., text: "Very nice, congrats!", like: true },
+  //   { id: ..., text: "Might've been waaaay better", like: false }
+  // ]
+```
+
+Finally, you want to relate these with your post, so you call `.attach()` passing the name of the **resource** (that can
+be either a **dataset** or a **fileset**) and filter criteria:
+
+```typescript
+  const firstPost = posts[0];
+  const commentsIds = comments.map(comment => comment.id);
+
+  await dom.dataset("posts")
```

Comment:
Don't you think it looks complex? 
Would be good to have opportunity to make :
.attach("comments", [
{ text: "bla bla bla"},
{ text: "do do do"}
])
Automatically make attachment by IDs
I see only 1 limitation for now. It will not be possible to make 2+ relations in 1 shot.

Reply:
Yes, that would be nice. I created a card for it and I plan to update these docs together with this change.